commit a9221f160c28301850d5368d283c751b18950f2a
Author: Eric Anholt <eric@anholt.net>
Date:   Mon Jan 25 14:33:50 2016 -0800

    drm/vc4: Drop error message on seqno wait timeouts.
    
    These ioctls end up getting exposed to userspace, and having normal
    user requests print DRM errors is obviously wrong.  The message was
    originally to give us some idea of what happened when a hang occurred,
    but we have a DRM_INFO from reset for that.
    
    Signed-off-by: Eric Anholt <eric@anholt.net>

diff --git a/drivers/gpu/drm/vc4/vc4_gem.c b/drivers/gpu/drm/vc4/vc4_gem.c
index dc3044d..a6fa63f 100644
--- a/drivers/gpu/drm/vc4/vc4_gem.c
+++ b/drivers/gpu/drm/vc4/vc4_gem.c
@@ -352,9 +352,6 @@ vc4_wait_for_seqno(struct drm_device *dev, uint64_t seqno, uint64_t timeout_ns,
 	finish_wait(&vc4->job_wait_queue, &wait);
 	trace_vc4_wait_for_seqno_end(dev, seqno);
 
-	if (ret && ret != -ERESTARTSYS)
-		DRM_ERROR("timeout waiting for render thread idle\n");
-
 	return ret;
 }
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1198_a9221f160c28301850d5368d283c751b18950f2a.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1198_a9221f160c28301850d5368d283c751b18950f2a.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
